pkg/connector/workspace.go

```diff
@@ -102,7 +116,7 @@ func (o *workspaceResourceType) Entitlements(ctx context.Context, resource *v2.R
 }
 
 func (o *workspaceResourceType) Grants(ctx context.Context, resource *v2.Resource, pt *pagination.Token) ([]*v2.Grant, string, annotations.Annotations, error) {
-	users, err := o.client.GetUsersContext(ctx)
+	users, err := o.client.GetUsersContext(ctx, slack.GetUsersOptionTeamID(resource.Id.Resource))
```

Comment:
If we're not an org client does this still work? Does passing the param where it's not needed "just work" ?

Reply:
yep it will ignore it :)